### ISSUE TITLE
need support nginx https for webpack+proxy single docker

### ISSUE THREAD

**Issue by Xiaohua-Shen** [kubernetes-sigs/kui#4272]
**need support nginx `https` for webpack+proxy single docker**
below steps can bring up a local container with http mode
```
npx kui-build-webpack
npx kui-build-docker-with-proxy
npx kui-run-cproxy
```
But now I need the nginx to provide `https` service. 
So I just modify the Dockerfile and build-docker.sh a little to let nignx has certificate and listen on ssl. 

The proxy can startup, but when doing /exec request, it return below errors.
```
26.ecc216680af3bc15305e.bundle.js: 7 error establishing session undefined undefined Error: Failed to construct 'WebSocket': The URL's scheme must be either 'ws' or 'wss'. 'https' is not allowed.
    at c (81.ecc216680af3bc15305e.bundle.js:1)
```
<img width="578" alt="Screen Shot 2020-04-13 at 5 54 40 PM" src="https://user-images.githubusercontent.com/19463092/79111847-e3092e80-7daf-11ea-9bd1-a0396645ac45.png">

**Steps to reproduce the behavior**
The change I made for above test:
1. create nginx cert under conf.d directory
```
shenxh@shenxh:~/kui/packages/proxy$ ls conf.d/ssl
tls.crt  tls.key
```
2. modify conf.d/proxy.conf to use certs
```
  listen 80 ssl default_server;
  listen [::]:80 default_server;
  
  ssl_certificate /etc/nginx/ssl/tls.crt;
  ssl_certificate_key /etc/nginx/ssl/tls.key;
```

3. modify Dockerfile to include nginx certificate
```
COPY conf.d/ssl /etc/nginx/ssl
```

4. modify build-docker.sh to copy nginx certificate
```
function nginx {
    echo "nginx config"
    if [ ! -d "$BUILDDIR"/conf.d ]; then mkdir "$BUILDDIR"/conf.d; fi
    cp "$CLIENT_HOME"/node_modules/@kui-shell/proxy/conf.d/proxy.conf "$BUILDDIR"/conf.d/default.conf
    cp -R "$CLIENT_HOME"/node_modules/@kui-shell/proxy/conf.d/ssl "$BUILDDIR"/conf.d
}
```

**System**

- Kui version: current master branch
- Operating system: ubuntu 16.04

**Others**
I tried to start proxy with HTTPS mode too, but which is blocked by another issue https://github.com/IBM/kui/issues/4271
